src/ApplicationInsights.ServiceFabric.NugetSigning/ApplicationInsights.ServiceFabric.NugetSigning.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props" Condition="Exists('..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props')" />
```

Comment:
I might be mistaken, but I think for NuGet signing you need to use MicroBuild packages version 0.3.0. This is what we use for EventFlow and it seems to work.

Reply:
I verified 0.2.0 to be working by nuget verify.